src/main/java/com/KacFlor/ShopSpring/controller/WishlistController.java

```diff
@@ -22,8 +28,23 @@ public WishlistController(WishlistService wishlistService){
         this.wishlistService = wishlistService;
     }
 
+    @PreAuthorize("hasAuthority('" + Role.Fields.ADMIN + "')")
     @GetMapping
-    public List<Wishlist> getWishlist(){
-        return this.wishlistService.getWishlist();
+    public List<Wishlist> getAll(){
+        return this.wishlistService.getAll();
+    }
+
+    @PreAuthorize("hasAuthority('" + Role.Fields.ADMIN + "')")
+    @GetMapping("/{id}")
+    public ResponseEntity<Optional<Wishlist>> getById(@PathVariable("id") Integer id){
+        Optional<Wishlist> wishlist = Optional.ofNullable(wishlistService.getById(id));
+        return ResponseEntity.ok(wishlist);
+    }
+
+    @PreAuthorize("hasAnyAuthority('" + Role.Fields.USER + "', '" + Role.Fields.ADMIN + "')")
+    @GetMapping("/customer/{id}")
```

Comment:
I think you should take customer id as an option, not part of url

Reply:
this is endpoint for find wishlist by customer id. How can i set id of customer for this endpoint if i not put it into url?

Reply:
done in Fixes branch